parquet-variant/src/main/java/org/apache/parquet/variant/Variant.java

```diff
@@ -61,6 +61,14 @@ public Variant(ByteBuffer value, ByteBuffer metadata) {
     }
   }
 
+  public ByteBuffer getValueRawBytes() {
```

Comment:
I think this should have a different name. It is odd to return `ByteBuffer` from a method referring to "bytes". What about `getValueBuffer` and `getMetadataBuffer`?